test/EndToEnd/tests/ServicesTest.ps1

```diff
@@ -654,12 +654,12 @@ function Test-InstallPackageAPIInstalledLowerVersionPackage
     $p = New-ClassLibrary
 
     # Act
-    [API.Test.InternalAPITestHook]::InstallPackageApi("microsoft.owin","2.0.0")
-    [API.Test.InternalAPITestHook]::InstallPackageApi("microsoft.owin","3.0.0")
+    [API.Test.InternalAPITestHook]::InstallPackageApi("nuget.librarymodel","6.2.0")
+    [API.Test.InternalAPITestHook]::InstallPackageApi("nuget.librarymodel","6.3.0")
```

Comment:
this seems to have caused a test failure, as the target project is targeting net45, and is incompatible with the package.

Reply:
Yep, that seems like a true failure. I'm looking into it.

Reply:
I started with https://github.com/NuGet/NuGet.Client/commit/61713994c334ca0acc39dccdf2e08295cd39b1b0#diff-26005e27a6f30d3a28445c7720ff5a1ec00181c252735e4935b02b2b2afb58b7. 

I'll ping for a rereview once the tests are all passing.

Reply:
Interesting that cause binding redirect tests to start failing such as BindingRedirectClassLibraryWithDifferentDependents, BindingRedirectClassLibraryWithDifferentDependents and BindingRedirectDuplicateReferences. 

All 3 are failing as in https://github.com/NuGet/Home/issues/12292.

Reply:
I'll split the 2 templates. This might give us a hint about the binding redirect failures.

Reply:
I'll split the 2 templates. This might give us a hint about the binding redirect failures.

Reply:
Create separate templates. We already have a precedent with a 46 template.

Reply:
passing now